core/src/tvu.rs

```diff
@@ -292,12 +292,14 @@ impl Tvu {
         };
 
         let (voting_sender, voting_receiver) = unbounded();
+        // TODO: use validator configuration to compute value for use_quic_client parameter
```

Comment:
Is this PR blocked on this TODO or was the plan to merge with it intact?  If the latter, please replace with a Github issue

Reply:
I am doing some testing on testnet. Some changes will be needed in the PR based on that.
I'll update the PR, and file the issue if the TODO cannot be removed.

Reply:
sg! :shipit: